CorsixTH/Lua/dialogs/resizables/options.lua

```diff
@@ -248,7 +266,7 @@ end
 function UIOptions:selectResolution(number)
   local res = self.available_resolutions[number]
 
-  local callback = --[[persistable:options_resolution_callback]] function(width, height)
+  local callback = function(width, height)
```

Comment:
Did you verify this function was never persisted?

Either way this change belongs in its own commit.